src/app/cli/src/app.rs

```diff
@@ -197,9 +197,7 @@ pub async fn run(workspace_layout: WorkspaceLayout, args: cli::Cli) -> Result<()
             .build();
         let maybe_server_catalog = if cli_commands::command_needs_server_components(&args) {
             let server_catalog =
-                configure_server_catalog(&final_base_catalog, is_multi_tenant_workspace)
-                    .add_value(current_account.to_current_account_subject())
-                    .build();
+                configure_server_catalog(&final_base_catalog, is_multi_tenant_workspace).build();
```

Comment:
There should not be a need in "multi-tenancy" marker for server catalog

Reply:
Removed (by rebasing on your changes) -- Done